src/components/Coindetail.js

```diff
@@ -0,0 +1,65 @@
+import { useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+const Detail = () => {
+  const coinsState = useSelector((state) => state.cryptoCoin);
+  const cryptoCoin = coinsState.filter((coin) => (coin.show === true));
+  return (
+    <>
+      <Link to="/">
+        <p className="exit-div">
+          <i
+            className="Exit"
+            role="button"
+            label="next"
+            tabIndex={0}
+          />
+        </p>
+        <div><h1 className="logo">CoinX</h1></div>
+      </Link>
+      <table>
+
+        <div className="crypto-div">
+          {cryptoCoin.map((coin) => (
```

Comment:
- [ ] [OPTIONAL] Even though this was not mentioned by the previous reviewer, it would be good to stick to the project requirements theme colors which are either pink or light blue. **Even though I labeled this as optional I strongly suggest you make the changes**
- [ ] [OPTIONAL] Also, it would be good to make the section look tabular as it is provided in the Behance design.

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/206866540-522b451f-6b08-4cec-add6-7971b79073a1.png)

**You could also use the light blue theme color as well**